build/images/base/Dockerfile.controller

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 Antrea Authors
```

Comment:
I don't get why we need the whole file. controller has no dependency on any of the following.

Reply:
we are using this to build the base image to be used in the final controller image build.